src/defaultsettings.cpp

```diff
@@ -199,63 +203,59 @@ void set_default_settings(Settings *settings)
 	settings->setDefault("fallback_font_shadow", "1");
 	settings->setDefault("fallback_font_shadow_alpha", "128");
 
-	std::stringstream fontsize;
-	fontsize << TTF_DEFAULT_FONT_SIZE;
+	std::string font_size_str = to_string(TTF_DEFAULT_FONT_SIZE);
```

Comment:
Actually `TOSTRING()` won't work, because `*DEFAULT_FONT_SIZE` are defined with parenthesis around their values and that's included in the preprocessor-generated string.

The definitions could probably be changed though, I can't think of any unexpected behavior that would result from that.